src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,48 @@
+.calendar {
+  width: $width-of-calendar;
+  padding: 10px;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+
+  &__day {
+    @include square($size);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background: #eee;
+    color: black;
+    border: 1px solid black;
+    transition: background-color 0.5s, transform 0.5s;
+
+  }
+
+  @for $index from 1 through 31 {
+    & :nth-child(#{$index})::before {
+      content: "#{$index}";
+    }
+  }
+
+  @for $index from 28 through 31 {
+    &--month-length-#{$index} :nth-child(n+#{$index+1}) {
+      display: none;
+    }
+  }
+
+  @each $day, $margin in $day-of-week {
+    &--start-day-#{$day} :first-child {
+      margin-left: $margin;
+    }
+  }
+
+  & :nth-child(n):hover {
+    cursor: pointer;
+    background-color: pink;
+    transform: translateY(-20px);
+
+  }
+}
+
+.page__calendar {
```

Comment:
It might be in another file? (3. Checklist)

Reply:
True, fix it. Each block should be in separate file

Reply:
ok